README.rst

```diff
@@ -40,7 +40,7 @@ Create a new virtual environment::
 
 Install the prerequisites::
 
-    $ pip install -r requirements/base.txt
+    $ pip install --no-binary django -r requirements/base.txt -r requirements/local.txt
```

Comment:
Last thing, you can just replace `base.txt` with `local.txt` since `local` sources `test` which sources `base`.